src/system/data-import-product-images.md

```diff
@@ -48,7 +48,7 @@ Before you try to import a large amount of images, review the steps in the impor
     ![Data import images file directory]({% link images/images/data-import-images-file-to-import.png %}){: .zoom}
     _Images File Directory_
 
-    If importing multiple images for a single `sku`, insert a blank row below the `sku` and enter the additional image file names in the appropriate columns. The additional rows are understood to belong to the parent `sku`.
+    If importing multiple images for a single `sku`, insert the images in a column named `additional_images` (add the column if not already added), separated by commas. Example: `image02.jpg,image03.jpg`.
```

Comment:
I would remove the period after the example so users don't emulate. Unless of course it's required...your call.

```suggestion
    If importing multiple images for a single `sku`, insert the images in a column named `additional_images` (add the column if not already added), separated by commas. Example: `image02.jpg,image03.jpg`
```